exporter/kafkaexporter/config.go

```diff
@@ -64,6 +67,14 @@ type Config struct {
 	Authentication kafka.Authentication `mapstructure:"auth"`
 }
 
+type HeaderInjection struct {
+	// Whether to inject headers or not, default is false.
+	InjectHeaders bool `mapstructure:"inject_headers"`
+
+	// list of header complaint with kafka receiver
```

Comment:
What are the compliant headers? How does a user know which headers are compliant?

Reply:
```suggestion
	// headers to inject in kafka messages
```